mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -146,68 +148,61 @@ def format_sparql_query(title, year):
     return QUERY % {"Title": title, "Year": year}
 
 
-def wiki_query(data_csv, user_agent):
+def wiki_query(movie: MovieData, user_agent: str) -> EnrichedMovieData | None:
     """
-    Formats SPARQL query for Wiki data
+    Queries Wikidata for information about a movie.
 
     Parameters:
-    data_csv (list of lists): Rows of movie data with [movie ID, release year, title].
-    user_agent (str): used to identify our script when sending requests to Wikidata SPARQL API.
+    movie (MovieData): A MovieData object to use in the sparql query.
+    user_agent (str): Used to identify our script when sending requests to Wikidata SPARQL API.
 
     Returns:
-        list of WikiMovieData: A list of movieData instances with movie IDs, genres, and directors.
+    EnrichedMovieData: An EnrichedMovieData object containing information about the movie.
```

Comment:
I completely forgot to mention this, but the fact that `wiki_query` was actually `do_all_the_wiki_queries` and returned an entire list, has bugged me for some time. Thanks for fixing this!